packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -169,6 +169,11 @@ export class WaferMap<
     @observable public dies: WaferMapDie[] = [];
     @observable public diesTable: Table<T> | undefined;
 
+    /**
+     * @internal
+     */
+    @observable public diesTableIPC: Uint8Array | undefined;
```

Comment:
@rajsite and I talked through these changes a bit, and we both feel that a slight change of direction may be in order. First, it seems like we can do away with this new API, and handle the deserialization in the JS module file in Blazor. To do this, we will need to expose the necessary Apache APIs via the `WaferMap` type. The solution we discussed would involve creating a static member on `WaferMap` that exposed the `tableFromIPC` API. Like the following:
```
class WaferMap {
    ...
    public static Apache = {
        tableFromIPC: tableFromIPC;
    };
    ...
}
```
Then, in `NimbleBlazor.lib.module.js` you could do something like this:
```
WaferMap: {
        setDiesTable: async function (waferMapReference, data) {
           const diesTableIPC = window.customElements.get('nimble-wafer-map').Apache.tableIPC;
           const table = diesTableIPC(data);
           await waferMapReference.setData(table);
        }
        ...
}
```
Note the `setData` API on the component. This is the other major suggestion we think is prudent to do. This API accomplishes a few things:
- It consolidates the web component API, which makes it less confusing
- It mirrors the pattern we already have in place for the `Table`
- We can now synchronously validate the data within the `setData` implementation
- Because it is asynchronous, you can now await pending updates before processing the new data.

Reply:
Sounds like a good summary of what we discussed. Just updated the example to call the static `Arrow` instead of `Apache`